.gitignore

```diff
@@ -37,6 +37,10 @@ CMakeCache.txt
 
 leveldb*/
 NuRaft*/
+curl*/
+jsoncpp*/
+ethash*/
+lua*/
```

Comment:
These should be here IMO, because these 3PC dependencies should not be in source control - just like the NuRaft and LevelDB deps in 2PC/Atomizer